SheepDash/core/src/com/missionbit/game/sprites/Sheep.java

```diff
@@ -0,0 +1,37 @@
+package com.missionbit.game.sprites;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Rectangle;
+import com.badlogic.gdx.math.Vector3;
+
+/**
+ * Created by MissionBit on 6/27/17.
+ */
+
+public class Sheep {
```

Comment:
This used elsewhere?  If yes, would you want an abstract class or interface for the animal?